tests/__init__.py

```diff
@@ -1,6 +1,30 @@
+import ast
 import pytest
+import sys
 from vulture import Vulture
 
+HAS_ASYNC = hasattr(ast, 'AsyncFunctionDef')
+PY3 = sys.version_info >= (3, )
+PY2 = sys.version_info < (3, )
```

Comment:
No need to have these three global variables. Move the code into the functions, please. We don't even need local variables.